### DISCUSSION TITLE
Height of text input field?

### DISCUSSION THREAD

**Discussion by gvlx** [Budibase/budibase#3398] [Help] [answered]
Hello,

Do you know how the height of the Text and Rich Text input fields can be controlled?

I've raise this request #3375 already but maybe there's a way to do it anyway.

Thanks

**Comment by mjashanks** [comment, marked as answer]
There is no way to do it. The Rich text component needs some love - we will get to this soon enough, I hope: https://github.com/Budibase/budibase/issues/2967

In that issue, I also suggest that the standard Text Input should have a "Multi Line" (i.e. Text Area).

It is important that we do this, I'm just not sure when.
